Template/azuredeploy.json

```diff
@@ -227,6 +234,14 @@
                         {
                             "name": "LOG_ANALYTICS_WORKSPACE_KEY",
                             "value": "[if(parameters('useAzureMonitorOnEdge'), listKeys(resourceId('Microsoft.OperationalInsights/workspaces', variables('logAnalyticsName')), '2021-06-01').primarySharedKey, '')]"
+                        },
```

Comment:
Could we here put a generic variable "CONFIG_LOCATION"  set to concat('https://raw.githubusercontent.com/',variables('gitUsername'),'/iotedge-lorawan-starterkit/',variables('gitBranch'),'/Tools/Cli-LoRa-Device-Provisioning/DefaultRouterConfig/') and then build the file name with the region?

Reply:
hmmm thinking again that might need another variable, "ConcentratorRegion" as it might differ from the Azure deployment region. I think it's ok like this